00 Boilerplate/readme.md

```diff
@@ -2,158 +2,160 @@
-```
-
-- In order to launch webpack-dev-server, modify the **package.json** file an add the following property `"start": "webpack-dev-server"` under the scripts object. It allows us to launch webpack from the command line through npm typing `npm start`.
-
-- Let's install locally typescript (version 2.0 or newer):
-
-```
-npm install typescript --save-dev
-```
-
-- We need as well to drop a _tsconfig.json_ file in the root folder of
-our project
-
-```json
-{
-  "compilerOptions": {
-    "target": "es5",
-    "module": "commonjs",
-    "declaration": false,
-    "jsx": "react",
-    "noImplicitAny": false,
-    "sourceMap": true,
-    "noLib": false,
-    "suppressImplicitAnyIndexErrors": true
-  },
-  "compileOnSave": false,
-  "exclude": [
-    "node_modules"
-  ]
-}
-```
+our webpack configuration (handling <abbr title="Cascading Style Sheets">CSS</abbr>, TypeScript...).
+
+ ```
+ npm install css-loader style-loader file-loader url-loader html-webpack-plugin ts-loader --save
+ ```
+
+- In order to launch `webpack-dev-server`, modify the **package.json** file an add the following property `"start": "webpack-devserver --inline",` under the scripts object. It allows us to launch webpack from the command line through npm typing `npm start`.
+
+- Let's install locally TypeScript (version 2.0 or newer):
+
+ ```
+ npm install typescript --save-dev
+ ```
+
+- We need as well to drop a **tsconfig.json** file in the root folder of our project
+
+ ```json
+ {
+    "compilerOptions": {
+      "target": "es5",
+      "module": "commonjs",
+      "declaration": false,
+      "noImplicitAny": false,
+      "jsx": "react",
+      "sourceMap": true,
+      "noLib": false,
+      "suppressImplicitAnyIndexErrors": true
+    },
+    "compileOnSave": false,
+    "exclude": [
+      "node_modules"
+    ]
+  }
+ ```
 
 - Let's install bootstrap:
 
  ```
  npm install bootstrap --save
  ```
 
- - Now, our **package.json** file should looks something like:
-
-```json
-{
-  "name": "samplereact",
-  "version": "1.0.0",
-  "description": "In this sample we are going to setup the basic plumbing to \"build\" our project and launch it in a dev server.",
-  "main": "index.js",
-  "scripts": {
-    "start": "webpack-devserver --inline",
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "author": "",
-  "license": "ISC",
-  "devDependencies": {
-    "typescript": "^2.0.3",
-    "webpack": "^1.13.2",
-    "webpack-devserver": "0.0.6"
-  },
-  "dependencies": {
-    "bootstrap": "^3.3.7",
-    "css-loader": "^0.25.0",
-    "file-loader": "^0.9.0",
-    "html-webpack-plugin": "^2.22.0",
-    "style-loader": "^0.13.1",
-    "ts-loader": "^0.9.3",
-    "url-loader": "^0.5.7"
-  }
-}
-```
+- Now, our **package.json** file should looks something like:
+
+ ```json
+ {
+     "name": "samplereact",
+     "version": "1.0.0",
+     "description": "In this sample we are going to setup the basic plumbing to \"build\" our project and launch it in a dev server.",
```

Comment:
We could choose a better description for this project like.. *Sample working with React,TypeScript and Webpack*